test/extended/node/cgroups.go

```diff
@@ -0,0 +1,81 @@
+package node
+
+import (
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-node][Serial][Feature:Remove support to configure Cgroup v1 from OCP version >= 4.19]", func() {
```

Comment:
```suggestion
var _ = g.Describe("[sig-node][Serial][Feature:Remove support to configure Cgroup v1 from OCP version >= 4.19][apigroup:operator.openshift.io]", func() {
```
You could add the `[apigroup:operator.openshift.io]` tag to exclude the test suite for microshift jobs which does not siutable.